packages/react/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -24,13 +24,13 @@ export interface DropdownItemProps extends UIComponentProps<DropdownItemProps> {
   accessibilityItemProps?: any
 
   /** Item's content. */
-  content?: ShorthandValue
+  content?: ShorthandValue<BoxProps>
 
   /** Item's header. */
-  header?: ShorthandValue
+  header?: ShorthandValue<ImageProps>
```

Comment:
Oops you switched the Image and box props :)

Reply:
Fixed ❤️